docs/saml.md

```diff
@@ -0,0 +1,29 @@
+# SAML SSO configuration
+
+SAML SSO is available for users of us.prairielearn.com and ca.prairielearn.com. Reach out to support@prairielearn.com to get it set up.
+
+## Required attributes
+
+PrairieLearn requires that SAML identity providers (IdPs) make three attributes available.
+
+### Name
+
+The full name of the user, e.g. "Joe Smith". This attribute is often named `displayName` or `urn:oid:2.16.840.1.113730.3.1.241`.
+
+### UID
+
+An identifier with an institution-specific suffix, e.g. "jsmith@example.edu". This attribute is often named `eppn`, `eduPersonPrincipalName`, or `urn:oid:1.3.6.1.4.1.5923.1.1.1.6`.
+
+This attribute is allowed to change. For instance, at many institutions, someone who changes their name will receive an updated identifier. The next time they log in to PrairieLearn, their UID will be updated to reflect the latest value from the IdP.
+
+Note that this will often look like an email, but does not currently need to be routable as such.
+
+### UIN
```

Comment:
Our notion of a UIN is often the most difficult to describe to people, so I'd appreciate extra attention here to make sure this is as clear as possible.

Reply:
Super clear to me! I appreciate how many different words you found for "must not change" 😂

Reply:
Ha, thanks, I borrowed from Shibboleth's description of identifiers: https://shibboleth.atlassian.net/wiki/spaces/CONCEPT/pages/928645231/NameIdentifiers